### ISSUE TITLE
Behance authentication 

### ISSUE THREAD

**Issue by oskargicast** [omab/python-social-auth#530]
I've created a behance app and seems they have changed their API because they no longer provide **SOCIAL_AUTH_BEHANCE_SECRET**. http://psa.matiasaguirre.net/docs/backends/behance.html

![selection_108](https://cloud.githubusercontent.com/assets/1572530/6321254/38dc1ade-bac4-11e4-9229-a7f0e76f1176.png)

If I use the client id as secret key I get this error:

```json
{
http_code: "400",
messages: [
    {
        type: "error",
        message: "Bad Request"
    }
]
}
```

**Comment by omab** [comment]
It seems that behance dropped OAuth2 support recently(?) in what looks like an API restructuring, their [blog post](http://blog.behance.net/dev/introducing-the-behance-api) from Sept 2014 says that OAuth2 would be introduced "soon" but so far it seems still broken.

I've created a request on their help desk https://help.behance.net/requests/143083 (not sure if that's public).

Meanwhile I've added a deprecated/broken notice in the docs http://psa.matiasaguirre.net/docs/backends/behance.html.

I hope they solve this silly change soon.
